### DISCUSSION TITLE
Issues with Medical

### DISCUSSION THREAD

**Discussion by Tetlys** [acemod/ACE3#9034] [Q&A]
Currently finding that wounds auto heal themselves over time with no explanation? 

How? Shoot someone
What? After a few seconds its asif an AI is healing them very quickly, patches wounds etc.

**Comment by PabstMirror** [comment]
I think if AI are friendly they might not wait at all before starting healing (e.g. blufor shooting blufor ai)

possibly helped with https://github.com/acemod/ACE3/pull/8242 but I'm not sure if friendlys can suppress?
